src/platform/qt/GDBController.h

```diff
@@ -28,6 +28,7 @@ Q_OBJECT
 public slots:
 	void setPort(ushort port);
 	void setBindAddress(const Address&);
+	void setWatchpointsBehavior(const int watchpointsBehaviorId);
```

Comment:
```suggestion
	void setWatchpointsBehavior(int watchpointsBehaviorId);
```

const does nothing useful here.

Though ideally this would be the actual enum, for signal/slot reasons I can see why you didn't do that. A lambda slot may be preferable.